### PR TITLE
make formatting match a lang file's formatting

### PR DESCRIPTION
**Pull request by SpaceClouds42**
I renamed all the keys so that they match the dot separated format of lang files. I did have a question about these two:
```json
-> list (permission: npc_list)
"taterzens.command.list": "Available Taterzens:",
"taterzens.command.list": "Show loaded Taterzens"
```
Which one should be called `.list` and what should the other really be called? From what i can tell, /npc list doesnt take any arguments..

### CODE REVIEW

**Comment by SpaceClouds42** [comment]
I also noticed this message that I couldn't find a corresponding command. ```json
  "taterzens.command.flag.changed": "Taterzen flag has been changed to %s.",
```

**Comment by samolego** [comment]
> I also noticed this message that I couldn't find a corresponding command.
> 
> ```json
>   "taterzens.command.flag.changed": "Taterzen flag has been changed to %s.",
> ```

This one is used in `/npc edit flags <flag>` (whether npc can be pushed, leashed (doesn't work :P) or jump-attack)

**Comment by samolego** [comment]
> I renamed all the keys so that they match the dot separated format of lang files. I did have a question about these two:
> 
> ```json
> -> list (permission: npc_list)
> "taterzens.command.list": "Available Taterzens:",
> "taterzens.command.list": "Show loaded Taterzens"
> ```
> 
> Which one should be called `.list` and what should the other really be called? From what i can tell, /npc list doesnt take any arguments..

Those could actually be merged into the same key. One is used in hover text, but basically means the same ...
I'd go with `"taterzens.command.list": "Available Taterzens",` (without `:` so it can be used in both cases)

File: en_us.json [hunk 1]
```diff
@@ -1,73 +1,65 @@
...
+  "taterzens.command.loaded": "Show loaded Taterzens",
+  "taterzens.command.flag.changed": "Taterzen flag has been changed to %s.",
+  "taterzens.command.config.success": "Config was reloaded successfully.",
+  "taterzens.command.rename.success": "Taterzen has been renamed to %s.",
+  "taterzens.error.invalid.url": "URL %s is malformed.",
+  "taterzens.error.invalid.text": "The text nbt you entered is invalid.",
+  "taterzens.error.404.detected": "No Taterzen was detected in front of you."
+  "taterzens.error.404.id": "No Taterzens with id %s were found.",
+  "taterzens.error.select": "You have to select Taterzen first.",
+  "taterzens.error.permission": "Permission level %s is higher than your permission level.",
```
Comment (samolego) [inline en_us.json:64]: Comma should be removed

Comment (SpaceClouds42) [reply en_us.json]: oops

**Review by samolego — APPROVED** [review]

**Review by SpaceClouds42 — COMMENTED** [review]
lol it appears i started a review, i meant to request one

**Comment by samolego** [comment]
Thank you so much!

**Pull request merged**